### DISCUSSION TITLE
Salmon quantification of isoforms of a single gene

### DISCUSSION THREAD

**Discussion by alhbiostat** [COMBINE-lab/salmon#846] [Q&A (bulk specific)]
Hi!
We are investigating a gene that appears to have complex cell-specific isoform usage using bulk cell-sorted RNASeq data, the aim being to compare relative isoform abundance across differing cell populations.

My question is simply, is it valid to use Salmon to quantify transcripts of a single gene, rather than transcriptome wide? I have built an index using only the sequences for 10 alternate isoforms of this gene, and the human genome as decoy. Obviously, only a very small percentages of fragments map to these transcripts, and the calculated TPM is misleading in the context of the small number of mapped reads. I am planning to use tximport with countsFromAbundance = "scaledTPM" to generate a count matrix to take forward for analysis - am I doing something questionable by quantifying transcripts in this way?

Any thoughts would be much appreciated!

**Comment by SteffanChristiansen** [comment]
Hi,

I am considering a similar procedure of a gene of interest. Did you ever compare the quantification with a transcriptome wide index? In that case, was it truly beneficial to use your approach?

Best,
Steffan
